.github/workflows/pypi_publish.yml

```diff
@@ -48,8 +59,122 @@ jobs:
           name: wheels-sdist
           path: ./wheelhouse/*.tar.gz
 
+  build-wheels:
+    name: 'Wheels: ${{ matrix.os }} ${{ matrix.arch }} ${{ matrix.python-version }}'
+    runs-on: ${{ matrix.os }}
+    continue-on-error: true
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+        python-version: ['3.9', '3.10', '3.11', '3.12', '3.13']
+        include:
+          - os: ubuntu-latest
+            arch: x86_64
+          - os: windows-latest
+            arch: AMD64
+          - os: macos-latest
+            arch: arm64
+
+    steps:
+    - name: Check out the repo
+      uses: actions/checkout@v4
+      with:
+        fetch-depth: 0  # get the non-merge commit for PRs
+        fetch-tags: true # include tags to get correct version from setuptools_scm
+
+    # Don't use conda to build pip wheels
+    # (it's going to be messy when you import the packages outside of conda)
+    - name: 🐍 Set up Python
+      uses: actions/setup-python@v5
+      with:
+        python-version: ${{ matrix.python-version }}
```

Comment:
This happens exclusively in Windows in h5py; I understand that `pypa/cibuildwheel` should deal with it internally?

Reply:
Yes, although `ci/cibw_before_all_windows.sh` contains python commands. I guess we can just conditionally install this on windows only.